src/components/Time.jsx

```diff
@@ -43,6 +65,12 @@ class Time extends React.Component {
         return TimeUtils.timeAgo(this.props.time, config)
     }
 
+    _refresh() {
+        this.refresh_timeout = setTimeout(() => {
+            this.setState(() => this.state)
+        }, DELAY)
```

Comment:
Isn't the intention that this component continually refreshes the time? If so I think we need to use `setInterval` rather than `setTimeout`.